app/views/sessions/new.html.erb

```diff
@@ -18,18 +18,32 @@
 
     <%= form_tag(auth_path(:magiclink), method: :post, id: 'login-email') do %>
       <fieldset class="govuk-fieldset">
-        <div class="govuk-form-group">
-          <span class="govuk-label-wrapper">
-            <label class="govuk-label govuk-label--m">alebo e-mailom</label>
-          </span>
-          <span class="govuk-hint">
-            Pošleme Vám jednoduchý odkaz na prihlásenie sa.
-          </span>
-          <%= email_field_tag :email, nil, class: 'govuk-input' %>
-        </div>
+        <% if @email_error %>
+          <div class="govuk-form-group govuk-form-group--error">
```

Comment:
Toto by sa podla mna dalo vybavit inline `if`om, nie?
Napr
```ruby
<div class="govuk-form-group <%= "govuk-form-group--error" if @email_error.present? %>">
```

Reply:
@eronisko v tom if-e a els-e je niekolko riadkov HTML kodu, nemyslim teda ze inline if tu moze byt.

Reply:
Chapem. Toto by bolo asi najlepsie riesene custom componentom do ktoreho len hodis `error: @email_error`, ale tu to mozeme zatial nechat takto.